Sources/StripeKit/Products/Products/Product.swift

```diff
@@ -44,6 +44,8 @@ public struct Product: Codable {
     public var updated: Date?
     /// A URL of a publicly-accessible webpage for this product.
     public var url: String?
+    /// The marketing feature name. Up to 80 characters long.
+    public var marketingFeatures: [MarketingFeature]
```

Comment:
This should be optional since it's a `nullable` field from Stripe's side. Then we could add `nil` as a default parameter in the `init`.

Reply:
I think we can leave `marketingFeatures` as is right now because it's explicitly initialized as an empty array here https://github.com/vapor-community/stripe-kit/pull/268/files#diff-15e20f3d38da5a505a3767001e026a9838fa218fc5b8661795e73849ee9f65f7R67

Reply:
cc @Andrewangeta

Reply:
It needs to be optional because when it's decoded from JSON using a `Decoder` it won't default to an empty array of the value is `null`